package.json

```diff
@@ -5,9 +5,9 @@
   "main": "index.js",
   "scripts": {
     "lint": "yarn run eslint && yarn run mdlint",
-    "eslint": "eslint --ext .js --ext .jsx stories components",
+    "eslint": "eslint --ext .js --ext .jsx stories components tests",
     "mdlint": "markdownlint \"**/*.{md,mdx}\" --ignore \"node_modules/**\"",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```

Comment:
We should probably at least discuss adding the `--cache` flag here to help with performance.

Reply:
cache is enabled by default in the newer jest's

Reply:
Ah nice! That's awesome.